pkg/oci/load_balancer.go

```diff
@@ -201,9 +202,10 @@ func (cp *CloudProvider) readSSLSecret(svc *v1.Service) (string, string, error)
 // balancer, if it doesn't already exist.
 func (cp *CloudProvider) ensureSSLCertificate(ctx context.Context, lb *loadbalancer.LoadBalancer, spec *LBSpec) error {
 	name := spec.SSLConfig.Name
+	logger := cp.logger.With("certificateName", name, "loadbalancerName", *lb.DisplayName)
```

Comment:
I feel we should change this to always use ocid and not display names.

Reply:
I believe we always add the ocid as soon as we have it; just including display name as we often have that prior to getting an ocid. Thoughts?